ostree/ostree_dest.go

```diff
@@ -348,14 +379,24 @@ func (d *ostreeImageDestination) Commit() error {
 		return err
 	}
 
+	var selinuxHnd *C.struct_selabel_handle
+
+	if os.Getuid() == 0 {
+		selinuxHnd = C.selabel_open(C.SELABEL_CTX_FILE, nil, 0)
+		if selinuxHnd == nil {
+			errors.Errorf("cannot open the SELinux DB")
+		}
+		defer C.selabel_close(selinuxHnd)
```

Comment:
Probably does not matter, but this should be 
else defer

Reply:
I just realized I forgot the return at line 387.  Going to push a new version with that fixed and the defer in an else branch